packages/client/modules/userDashboard/containers/OrgBilling/OrgBillingRoot.tsx

```diff
@@ -20,18 +21,31 @@ interface Props {
 }
 
 const OrgBillingRoot = ({organization}: Props) => {
-  const atmosphere = useAtmosphere()
+  const queryRef = useQueryLoaderNow<OrgBillingRootQuery>(orgBillingRootQuery, {
+    orgId: organization.id,
+    first: 3
+  })
   return (
-    <QueryRenderer
-      environment={atmosphere}
-      query={query}
-      variables={{orgId: organization.id, first: 3}}
-      fetchPolicy={'store-or-network' as any}
-      render={renderQuery(OrgBilling, {props: {organization}, size: LoaderSize.PANEL})}
-    />
+    <Suspense fallback={''}>
+      {queryRef && <OrgBillingContainer queryRef={queryRef} organization={organization} />}
+    </Suspense>
   )
 }
 
+interface OrgBillingContainerProps {
+  queryRef: PreloadedQuery<OrgBillingRootQuery>
+  organization: OrgBillingRoot_organization
+}
+
+function OrgBillingContainer(props: OrgBillingContainerProps) {
```

Comment:
This pattern would be simpler without involving modifications to the child component, e.g. `OrgBilling` (especially if it involves the `createPaginationContainer` and `createRefetchContainer` HOC components)

Because it uses `usePreloadedQuery` to request data, I call it the `Container` of component (compare to `Wrapper`)

Reply:
makes sense, I'll backlog that work in https://github.com/ParabolInc/parabol/issues/6431